tests/robustness/model/deterministic.go

```diff
@@ -377,6 +389,13 @@ type EtcdLease struct {
 	Keys    map[string]struct{}
 }
 
+func (el EtcdLease) DeepCopy() EtcdLease {
+	return EtcdLease{
```

Comment:
Would like to rewrite leases as they are overcomplicated, we have max 20 keys and even less leases. Using so many maps here is an overkill.